test/EvolvingWrapperTestCase.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace PhoneBurnerTest\Http\Message;
+
+use PHPUnit\Framework\TestCase;
+
+abstract class EvolvingWrapperTestCase extends WrapperTestCase
+{
+    abstract public function provideWithMethods(): \Generator;
```

Comment:
SHOULD: Instead of `Generator`, we probably should just specify a return type declaration of `iterable` here, since the data provider just needs an array of arrays or a `Traversable` that returns arrays.